frontend/src/components/Form.js

```diff
@@ -41,11 +41,41 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Validates from inputs on submission and alerts user on error
+ * @return {Boolean} true or false based on validity of inputs
+ * @param {String} role
+ * @param {String} productArea
+ * @param {Number} duration
+ * @param {Number} timeAvailableUntilMilliseconds
+ * @param {Number} currentTimeMilliseconds
+ */
+export function validateFormInputs(role, productArea, duration,
+    timeAvailableUntilMilliseconds, currentTimeMilliseconds) {
```

Comment:
nit: if all the parameters can't fit on one line, put each one on its own line

Reply:
Done, thanks!